modules/solid_mechanics/test/tests/eigen_decomp_material/tests

```diff
@@ -1,13 +1,12 @@
 [Tests]
   design = 'source/materials/EigenDecompositionMaterial.md'
   issues = '#29179'
-  requirement = 'The system shall output an eigen decomposition of a symmetric rank two tensor '
```

Comment:
This would have worked if you had added one more level to set up a group of tests.

Reply:
I was going to but I didn't know what to name it...... [Tests][tests]

Reply:
[Tests]
  design = 'source/materials/EigenDecompositionMaterial.md'
  issues = '#29179'
  [eigen_decomp_output]
     [this case]
      ...
      [this other one]

    []
  []
```